src/site/stages/build/plugins/create-feature-toggles.js

```diff
@@ -0,0 +1,36 @@
+const {
+  FlipperClient,
+} = require('../../../../platform/utilities/feature-toggles/flipper-client.js');
+const environmentsConfigs = require('../../../constants/environments-configs');
+const environments = require('../../../constants/environments');
+
+function createFeatureToggles(buildOptions) {
+  const environmentName = buildOptions.buildtype;
+
+  return async (files, metalsmith, done) => {
+    const { fetchToggleValues } = FlipperClient({
+      host: environmentsConfigs[environmentName].API_URL,
+    });
+
+    let toggleValues;
+    try {
+      toggleValues = await fetchToggleValues();
+    } catch (error) {
+      if (environmentName === environments.LOCALHOST) {
+        toggleValues = {};
+      } else {
+        const errorMessage = `createFeatureToggles failed: ${error.message}`;
```

Comment:
For now this is hitting the internet for our feature toggle values. This will fail the build if the feature toggle service is not available.